### PR TITLE
Add setup instructions to README

### PR DESCRIPTION
**Pull request by steverydz**
Done:
Add setup instructions for MAAS UI to the README

QA:
- Read through the instructions and confirm they are clear and make sense
- If you don't have MAAS UI setup locally try these instructions and check they work as expected

### CODE REVIEW

**Comment by steverydz** [comment]
@lilyvidenova Do you mean logs inside MAAS rather than maas.io?

File: README.md [hunk 1]
```diff
@@ -10,10 +10,47 @@ This is a mono repo for MAAS frontend projects.
...
+
+```
+cp ui/.env.development ui/.env.development.local
+```
+
+Update the contents of that file to:
+
+```
+REACT_APP_BASENAME="/MAAS/settings"
+REACT_APP_MAAS_URL="http://<your-local-maas-ip>:5240/MAAS",
```
Comment (squidsoup) [inline README.md]: that trailing comma should not be there

File: README.md [hunk 2]
```diff
@@ -10,10 +10,47 @@ This is a mono repo for MAAS frontend projects.
...
+
+From here you should be able to view the project at <your-local-maas-ip>:8000
 
 # Adding a new project
 
 To add a new project, edit `package.json` and add the project's directory name to the `workspaces` array.
 
 To import modules from existing projects in your new project, add the dependant projects to your projects dependencies in `package.json`. See [Yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) for details.
+
+# Setting up MAAS UI
```
Comment (squidsoup) [inline README.md]: Heading here, but no content?

**Comment by steverydz** [comment]
@squidsoup All fixed

**Pull request merged**